### ISSUE TITLE
Postgres cannot be downloaded from other than default place

### ISSUE THREAD

**Issue by janzyka** [yandex-qatools/postgresql-embedded#55]
We created our own packages and put them into our own Nexus repository. Unfortunately when embedded postgres starts it uses this nexus settings for the main (postgres) command but for the rest the defaults are used. This means for example that for "InitDb" command it tries to download the package again from the default location. Even though the package has already been downloaded from nexus. The config we have:

```
new RuntimeConfigBuilder().defaults(command)
            .artifactStore(new ArtifactStoreBuilder()
                .defaults(Command.Postgres)
                .download(new DownloadConfigBuilder()
                    .defaultsForCommand(command)
                    .downloadPath(postgrePackage.getNexusUrl() + "/service/local/repositories/" + postgrePackage.getRepository() + "/content/")
                    .packageResolver(packageResolver)
                    .build())
                .build())
            .build();
```

**Comment by smecsia** [comment]
It looks like a duplicate of #39

**Comment by janzyka** [comment]
I would say not exactly even though the root cause is the same. The common problem of both issues is that 2 of the 3 commands (initDb & createDb) use the default settings rather than settings passed to RuntimeConfigBuilder. In my case the problem is that these commands attempt re-download from default location where in #39 the problem shows as unable to change download directory.

**Comment by smecsia** [comment]
Fixed in 1.17.
